test/integration/base_integration_test.h

```diff
@@ -109,6 +109,35 @@ class BaseIntegrationTest : protected Logger::Loggable<Logger::Id::testing> {
   void registerPort(const std::string& key, uint32_t port);
   uint32_t lookupPort(const std::string& key);
 
+  /**
+   * @brief Schedule a callback on main thread to force create the traffic stats for the given
+   * cluster.
+   *
+   * @param cluster_name name of the cluster.
+   * @return true if cluster is found.
+   * @return false otherwise.
+   */
+  bool forceCreationOfClusterTrafficStats(absl::string_view cluster_name) {
+    // With https://github.com/envoyproxy/envoy/pull/23921 ClusterInfo::trafficStats is lazy init.
+    // We need to trigger creation of ClusterInfo::trafficStats() by calling the * operator.
+    absl::Notification notifier;
+
+    bool cluster_found = false;
+    test_server_->server().dispatcher().post([&]() {
+      const Upstream::ClusterConstOptRef& cluster_ref =
+          test_server_->server().clusterManager().clusters().getCluster(cluster_name);
+      if (cluster_ref.has_value()) {
+        const Upstream::ClusterTrafficStats& traffic_stats =
+            *cluster_ref->get().info()->trafficStats();
+        UNREFERENCED_PARAMETER(traffic_stats);
```

Comment:
Why not just write:
```
*cluster_ref->get().info()->trafficStats();
```
and drop the unused variable-name and unref'd param decl.

Reply:
I was trying to say: "look at this thing that has now been instantiated!". I guess that's not necessary. 

Done.